girder_annotation/girder_large_image_annotation/web_client/views/imageViewerWidget/geojs.js

```diff
@@ -120,6 +120,43 @@ var GeojsImageViewerWidgetExtension = function (viewer) {
             return numOverlays;
         },
 
+        /**
+         * Set additional parameters for pixelmap overlays.
+         * @param {object} layerParams An object containing layer parameters. This should already have
+         * generic properties for overlay annotations set, such as the URL, opacity, etc.
+         * @param {object} pixelmapElement A pixelmap annotation element
+         * @returns An object containing parameters needed to create a pixelmap layer.
+         */
+        _addPixelmapLayerParams(layerParams, pixelmapElement) {
+            // For pixelmap overlays, there are additional parameters to set
+            layerParams.url = layerParams.url + `?encoding=PNG`;
+            let pixelmapData = pixelmapElement.values;
+            if (pixelmapElement.boundaries) {
+                let valuesWithBoundaries = new Array(pixelmapData.length * 2).fill(0);
+                pixelmapData = valuesWithBoundaries.map((d, i) => pixelmapData[Math.floor(i / 2)]);
```

Comment:
You aren't using the values in the valuesWithBoundaries array.  So `fill` is a wasted operation.  This would probably be better as:
```
let valuesWithBoundaries = new Array(pixelmapData.length * 2);
for (let i = 0; i < pixelmapData.length; i += 1) {
  valuesWithBoundaries[i * 2] = valuesWithBoundaries[i * 2 + 1] = pixelmapData[i];
}
```
Function calls tend to be slow, so avoiding `map`ping something that is potentially millions long is preferred.